Servers/UI/OJS.Servers.Ui/ClientApp/src/components/guidelines/trees/Tree.tsx

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import {TreeView, TreeItem} from "@material-ui/lab";
+import {MdChevronRight, MdExpandMore} from "react-icons/md";
+import {isArray} from "lodash";
+
+import styles from './Tree.module.scss';
+
+interface ITreeItemType {
+    id: string,
+    name: string,
+    children?: ITreeItemType[],
+}
+
+interface ITreeProps {
+    items: ITreeItemType[],
+    handleTreeItemClick: (node: ITreeItemType) => void,   
```

Comment:
`on...` for props
`handle...` for handlers.

This is prop, so... `onTreeItemClick`